frame/social-swap2/src/lib.rs

```diff
@@ -192,10 +192,10 @@ decl_module! {
             ensure!(to != token0 && to != token1, Error::<T>::InvalidTo);
 
             if amount0_out > 0u32.into() {
-				T::FungibleToken::transfer(&social_token_id, &token0, &to, amount0_out);
+				T::FungibleToken::transfer(&social_token_id, &token0, &to, amount0_out)?;
             }
             if amount1_out > 0u32.into() {
-				T::FungibleToken::transfer(&social_token_id, &token1, &to, amount1_out);
+				T::FungibleToken::transfer(&social_token_id, &token1, &to, amount1_out)?;
             }
             // TODO:
             // if (data.length > 0) IUniswapV2Callee(to).uniswapV2Call(msg.sender, amount0Out, amount1Out, data);
```

Comment:
it can remove because `data.length` equal `0` usually (https://uniswap.org/docs/v2/smart-contracts/pair/#swap-1)